tools/js2c.py

```diff
@@ -272,8 +276,13 @@ def AddModule(module, source):
     if name.endswith('.gypi'):
       # Currently only config.gypi is allowed
       assert name == 'config.gypi'
-      lines = re.sub(r'\'true\'', 'true', lines)
-      lines = re.sub(r'\'false\'', 'false', lines)
+      config_filename = str(name).replace('gypi', 'json')
```

Comment:
Why not just pass config.json to js2c in node.gyp?

Reply:
For the time being I'm assuming 1:1 correspondence, and I want to keep the embedded `config_raw` data exactly the same using:
https://github.com/nodejs/node/blob/c663277042ed56a221bfbb25b5acf18a1ce2fdbd/tools/js2c.py#L285
So the tight coupling is there anyway...